### DISCUSSION TITLE
Cara setting atau melihat password di GitHub Mobile?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1603] [Q&A] [answered]
Tolong sertakan screenshot

**Comment by realodix** [comment]
Di APK?

Maksudnya Github Mobile?

**Comment by Jokopentil** [comment]
Ya

Pada tanggal Min, 22 Okt 2023 19.07, Kei ***@***.***> menulis:

**Comment by realodix** [comment, marked as answer]
Baca dokumentasinya saja di https://docs.github.com/en/get-started/using-github/github-mobile

atau bisa ikut berdiskusi di https://github.com/orgs/community/discussions/categories/mobile
